test/jdk/java/awt/print/PrinterJob/PrintTextTest.java

```diff
@@ -40,36 +42,32 @@
 
 public class PrintTextTest extends Component implements Printable {
```

Comment:
Please expand wild card imports and update the license copyright year.

`* Copyright (c) 2007, 2024, Oracle and/or its affiliates.`

Reply:
Thanks for the feedback -- done.